HW_2/Task_2.py

```diff
@@ -0,0 +1,28 @@
+#Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем. 
+#Программа должна возвращать сумму и произведение* дробей. Для проверки своего кода используйте модуль fractions.
+import math
+
+def fraction():
+    frst: str = input("Введите первую дробь в формате x/y (целые числа):")
+    scnd: str = input("Введите вторую дробь в формате x/y (целые числа):")
+    action: str = input("Введите маь. операцию (сложение - + или умножение - *):")
+    a = frst.split("/")
+    b = scnd.split("/")
+    if action == "+":
+        c:int  = math.lcm(int(a[1]),int(b[1]))
+        d:int = int(a[0])*(c/int(a[1]))
+        e:int = int(b[0])*(c/int(b[1]))
```

Comment:
Не забывайте приводить к PEP8 и следите за аннотацией типов.
```suggestion
        c: int = math.lcm(int(a[1]), int(b[1]))
        d: float = int(a[0]) * (c / int(a[1]))
        e: float = int(b[0]) * (c / int(b[1]))
```